diff --git a/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts b/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
index 89c2d5c..3353c17 100644
--- a/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
+++ b/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
@@ -15,6 +15,20 @@ export declare class FederatedPointerEvent extends FederatedMouseEvent implement
      * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/width
      */
     width: number;
+    /**
+     * The angle in radians of a pointer or stylus measuring the vertical angle between
+     * the device's surface to the pointer or stylus.
+     * A stylus at 0 degrees would be directly parallel whereas at π/2 degrees it would be perpendicular.
+     * @see https://developer.mozilla.org/docs/Web/API/PointerEvent/altitudeAngle)
+     */
+    altitudeAngle: number;
+    /**
+     * The angle in radians of a pointer or stylus measuring an arc from the X axis of the device to
+     * the pointer or stylus projected onto the screen's plane.
+     * A stylus at 0 degrees would be pointing to the "0 o'clock" whereas at π/2 degrees it would be pointing at "6 o'clock".
+     * @see https://developer.mozilla.org/docs/Web/API/PointerEvent/azimuthAngle)
+     */
+    azimuthAngle: number;
     /**
      * The height of the pointer's contact along the y-axis, measured in CSS pixels.
      * radiusY of TouchEvents will be represented by this value.
