diff --git a/node_modules/@vitest/runner/dist/types.d.ts b/node_modules/@vitest/runner/dist/types.d.ts
index 787f7b5..f94d800 100644
--- a/node_modules/@vitest/runner/dist/types.d.ts
+++ b/node_modules/@vitest/runner/dist/types.d.ts
@@ -9,26 +9,26 @@ import '@vitest/utils';
 interface VitestRunnerConfig {
 	root: string;
 	setupFiles: string[];
-	name?: string;
+	name?: string | undefined;
 	passWithNoTests: boolean;
-	testNamePattern?: RegExp;
-	allowOnly?: boolean;
+	testNamePattern?: RegExp | undefined;
+	allowOnly?: boolean | undefined;
 	sequence: {
-		shuffle?: boolean;
-		concurrent?: boolean;
+		shuffle?: boolean | undefined;
+		concurrent?: boolean | undefined;
 		seed: number;
 		hooks: SequenceHooks;
 		setupFiles: SequenceSetupFiles;
 	};
 	chaiConfig?: {
-		truncateThreshold?: number;
-	};
+		truncateThreshold?: number | undefined;
+	} | undefined;
 	maxConcurrency: number;
 	testTimeout: number;
 	hookTimeout: number;
 	retry: number;
-	includeTaskLocation?: boolean;
-	diffOptions?: DiffOptions;
+	includeTaskLocation?: boolean | undefined;
+	diffOptions?: DiffOptions | undefined;
 }
 /**
 * Possible options to run a single file in a test.
